target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -46,7 +46,8 @@ case "$board" in
 		;;
 	dlink,dap-1620-b1|\
 	dlink,dir-853-a1|\
-	dlink,dra-1360-a1)
+	dlink,dra-1360-a1|\
```

Comment:
This can be dealt with in dts.

Reply:
Note I've updated the commit message about 130acn 
it has only 1 chip mt7615DB and 1 active pcie: pcie0
I couldn't find any other devices in ramips_mt7621 with mt7615 that don't use `10_fix_wifi_mac` for this

I've made a some tries but I cannot configure it properly with only dts
stock mac address for wlan are:
xx:xx:xx:xx:xx:xa - phy0
xx:xx:xx:xx:xx:xb - phy1

if i add a macaddress as nvmem-cells it applies to both phy0 and phy1
`nvmem-cells = <&eeprom_factory_0>, <&macaddr_factory_4>;`

if i omit it, 
phy0 gets configured and works: xx:xx:xx:xx:xx:xb
while phy1 get a random first 3 byte and doesn't work: yy:yy:yy:xx:xx:xb 

since at least phy1 gets his correct mac, 
i've updated the function in /etc/hotplug.d/ieee80211/10_fix_wifi_mac
to overwrite only phy0

Reply:
here is how to do it in dts:

https://github.com/openwrt/mt76/commit/65cc3daf2a332cc658e9f7438cdadde4392e672e

And yes, all of those devices need to be migrated.

Reply:
fixed in last push

As a side note i couldn't manage to make wlan leds work properly, 
by specifying the right led_pin for each band.

case 1:
With the current dts, (led_pin = 2 for phy0, implicit led_pin = 0 for phy1) 
they work only in DBDC mode and they are inverted.
However this is fixable in usermode, by reversing the trigger
```
	echo phy1tpt > /sys/class/leds/mt76-phy0/trigger
	echo phy0tpt > /sys/class/leds/mt76-phy1/trigger
```

Doing it in 01_leds like:
```
	gemtek,wvrtm-130acn)
	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "mt76-phy0" "phy1tpt"
	ucidef_set_led_wlan "wlan5g" "WiFi 5GHz" "mt76-phy1" "phy0tpt"
	;;
```
Doesn't work for both because in default 'non dbdc mode'
phy1 doesn't came up at boot and mt76-phy0 results with trigger set to `none`

case 2:
If I don't declare a led_pin (or if set to 0) they both use the default (led_pin = 0)
Here, led of 2.4GHz works both in 'dbdc mode' and in 'non dbdc mode'
but both radio use the same led_pin, so the 5GHz led never turn on.

I think this would require being able to declare a led node for each band, like below:
but it seems not possibile to me with the current driver
```
		band@0 {
			...			
			led {
				led-sources = <0>;
			};
		};

		band@1 {
			...
			led {
				led-sources = <2>;
			};
		};
```

Reply:
I think the current solution is fine. if you find a real fix please provide it later.